README.md

```diff
@@ -1,159 +1,24 @@
 <img height="90" alt="Slonik Duke" align="right" src="docs/media/img/slonik_duke.png" />
 
-# PostgreSQL JDBC Driver
+# PostgreSQL JDBC Driver with adjustments for CrateDB
 
-PostgreSQL JDBC Driver (PgJDBC for short) allows Java programs to connect to a PostgreSQL database using standard, database independent Java code. Is an open source JDBC driver written in Pure Java (Type 4), and communicates in the PostgreSQL native network protocol.
+## About
+This is a fork of the vanilla [PostgreSQL JDBC Driver] to support specific
+details for [CrateDB], which can be inspected at [differences to the upstream
+pgJDBC]. It is used by the [CrateDB legacy JDBC driver] for covering certain
```

Comment:
Other than

> ... to support specific details for CrateDB, which can be inspected ...

it would be sweet to be able to add a more high-level summary here, similar to the "What's inside" section at [^1].

[^1]: https://github.com/crate/crate-jdbc/blob/159010bb29e3da5c90819e081167942a66a8ed2b/docs/index.rst#whats-inside

Reply:
I think a simple sentence like, to avoid certain statements and queries on internal PostgreSQL tables which `were` (or are? I don't know) not supported by CrateDB.

Reply:
Thanks, I've just updated the patch accordingly.